cmd/imagec/main.go

```diff
@@ -181,14 +181,16 @@ func main() {
 		log.SetOutput(io.MultiWriter(os.Stdout, f))
 	}
 
+	op := optrace.NewOperation(context.Background(), "imagec")
+
 	switch imageCOptions.operation {
 	case PullImage:
 		options := imageCOptions.options
 
 		options.Outstream = os.Stdout
 
 		ic := imagec.NewImageC(options, streamformatter.NewJSONStreamFormatter())
-		if err := ic.PullImage(); err != nil {
+		if err := ic.PullImage(op); err != nil {
 			log.Fatalf("Pulling image failed due to %s\n", err)
```

Comment:
It seems like it'd be useful to update the calls to `log.*` to use `op.*` in methods where you've introduced an `op`.